Cisco/cisco-wsa/ingest/parser.yml

```
@@ -0,0 +1,286 @@
+name: cisco-wsa
+pipeline:
+  - name: parsed_event
+    external:
+      name: grok.match
+      properties:
+        input_field: original.message
+        output_field: message
+        pattern: '^%{NUMBER:timestamp}\s+%{NUMBER:elapsed}\s+%{IP:source_ip}\s+%{WORD:code}/%{NUMBER:status}\s+%{NUMBER:http_response_bytes}\s+%{WORD:method}\s+%{NOTSPACE:url}\s+%{NOTSPACE:username}\s+%{NOTSPACE:hierarchy_code}/(%{IP:peerhostip}|%{NOTSPACE:peerhost})\s+%{NOTSPACE:mime_type}\s+%{WORD:decision_tag}-%{WORD:policy_group_name}-%{WORD:policy_identity}-%{WORD:outbound_malware_scanning_policy}-%{WORD:data_security_policy}-%{WORD:external_dlp_policy}-%{WORD:routing_policy}\s+<%{NOTSPACE:scanning_verdict_information}>\s+%{DATA:user_agent}$'
+  - name: parsed_timestamp
+    external:
+      name: date.parse
+      properties:
+        input_field: parsed_event.message.timestamp
+        output_field: start_date
+        format: timestamp
+  - name: parsed_scanning_verdict
+    external:
+      name: dsv.parse-dsv
+      properties:
+        input_field: '{{parsed_event.message.scanning_verdict_information}}'
+        output_field: results
+        columnnames:
+          - url_category
+          - url_reputation_score
+          - webroot_malware_category_code
+          - webroot_malware_name
+          - webroot_threat_risk_ratio
+          - webroot_threat_id
+          - webroot_trace_id
+          - mcafee_malware_category_code
+          - mcafee_file_name
+          - mcafee_detection_error
+          - mcafee_detection_type_code
+          - mcafee_malware_type_code
+          - mcafee_malware_name
+          - sophos_malware_category_code
+          - sophos_process_exit_code
+          - sophos_file_name
+          - sophos_malware_name
+          - cisco_data_security_code
+          - cisco_external_dlp_code
+          - predefined_url_category
+          - cisco_dca_url_category
+          - malware_category
+          - threat_type_code
+          - application_name
+          - application_type
+          - application_behavior
+          - safe_browsing_result
+          - bandwidth_avg
+          - bandwidth_was_throttled
+          - user_type
+          - malware_result
+          - outbound_malware_scanning_threat_name
+          - threat_category_code
+          - threat_name
+          - threat_score
+          - analysis_is_required
+          - file_name
+          - file_sha256
+          - file_reputation
+          - archivescan_result
+          - archivescan_detail
+        delimiter: ','
+  - name: set_ecs_fields
+    external: null
+  - name: set_cisco_fields
+    external: null
```

I think those fields are not necessary when null (even is they are added by custom formats UI
```suggestion
  - name: set_cisco_fields
```

---

Applied